src/libraries/Microsoft.PowerFx.Core/Public/Engine.cs

```diff
@@ -47,11 +47,10 @@ public IEnumerable<string> GetAllFunctionNames()
         /// Create a resolver for use in binding. This is called from <see cref="Check(string, RecordType, ParserOptions)"/>.
         /// Base classes can override this is there are additional symbols not in the config.
         /// </summary>
-        /// <param name="alternateConfig">An alternate config that can be provided. Should default to engine's config if null.</param>        
         /// <returns></returns>
-        private protected virtual INameResolver CreateResolver(PowerFxConfig alternateConfig = null)
```

Comment:
>PowerFxConfig alternateConfig = null [](http://example.com/codeflow?start=63&length=36)

Great to remove this!
FYI, technically an internal breaking change - will require a trivia update in PAClient.

Reply:
Yeah, will pick up with #520